src/Keboola/DbExtractor/Extractor/Common.php

```diff
@@ -4,20 +4,24 @@
 
 namespace Keboola\DbExtractor\Extractor;
 
+use Keboola\DbExtractor\Exception\ApplicationException;
 use Keboola\DbExtractor\Exception\UserException;
 
 class Common extends Extractor
 {
+    /**
+     * @var  array
+     */
```

Comment:
single line? You can update your File templates 

![image](https://user-images.githubusercontent.com/642928/39519878-d2c29288-4e08-11e8-9a87-aa09c528f7e5.png)